replica/replica.go

```diff
@@ -962,6 +969,23 @@ func (r *Replica) createDisk(name string, userCreated bool, created string) erro
 	r.volume.UserCreatedSnap = append(r.volume.UserCreatedSnap, userCreated)
 	r.activeDiskData = append(r.activeDiskData, &newHeadDisk)
 
+	// assign it to temp variable, so that	r.info remains
+	// intact in case of crash
+	info := r.info
+	info.Head = newHeadDisk.Name
+	info.Dirty = true
+	info.Parent = newSnapName
+	info.RevisionCounter = newHeadDisk.RevisionCounter
```

Comment:
is RevisionCounter missed in old code?

Reply:
yes

Reply:
Does this revision counter play any role ??

Reply:
yes, check GetReplica in REST Api calls